mex/extractors/datenkompass/transform.py

```diff
@@ -401,14 +405,11 @@ def transform_resources(
             )
             beschreibung = "n/a"
             if item.description:
-                description_de = [
-                    fix_quotes(d.value) for d in item.description if d.language == "de"
-                ]
-                beschreibung = delim.join(
-                    description_de
-                    if description_de
-                    else [fix_quotes(d.value) for d in item.description]
-                )
+                beschreibung = delim.join(get_german_text(item.description))
+                beschreibung_soup = BeautifulSoup(beschreibung, "html.parser")
+                for a in beschreibung_soup.find_all("a", href=True):
+                    a.replace_with(a["href"])
```

Comment:
```suggestion
                beschreibung = delim.join(get_german_text(item.description))
                beschreibung_soup = BeautifulSoup(beschreibung, "html.parser")
                for a in beschreibung_soup.find_all("a", href=True):
                    a.replace_with(a["href"])
```
don't we need the same for the field in activities and bibliographic resources

Reply:
[666cc62](https://github.com/robert-koch-institut/mex-extractors/pull/65/commits/666cc62271fd6146d0ca6e6c3534d2511266955e)